AutomatedTesting/Gem/PythonTests/atom_renderer/atom_hydra_scripts/hydra_AtomMaterialEditor_BasicTests.py

```diff
@@ -3,12 +3,12 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
-
-import azlmbr.materialeditor will fail with a ModuleNotFound error when using this script with Editor.exe
-This is because azlmbr.materialeditor only binds to MaterialEditor.exe and not Editor.exe
-You need to launch this script with MaterialEditor.exe in order for azlmbr.materialeditor to appear.
 """
 
+# import azlmbr.materialeditor will fail with a ModuleNotFound error when using this script with Editor.exe
+# This is because azlmbr.materialeditor only binds to MaterialEditor.exe and not Editor.exe
+# You need to launch this script with MaterialEditor.exe in order for azlmbr.materialeditor to appear.
```

Comment:
this is an interesting one. The problem with having that as different executable means that the EditorTestSuite will not work with this because it launches and scripts Editor.exe, not MaterialEditor.exe.
Let's sync at some point to figure out how to fix this one

Reply:
sure, think an easy option will be passing the desired executable you want instead of hard-coding `Editor.exe` (that's what was done on LTT with the `generic_launcher`